spec/requests/api/microposts_api_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'swagger_helper'
+
+RSpec.describe 'api/microposts_api', type: :request do
+
+  setup do
+    @microposts = Micropost.all
+  end
+
+  path '/api/microposts' do
+    post('get all microposts data in json') do
+      tags 'Microposts'
+      produces 'application/json'
+      response(200, 'successful') do
```

Comment:
add something like it:
```      response '200', 'blog found' do
        schema type: :object,
          properties: {
            id: { type: :integer },
            title: { type: :string },
            content: { type: :string }
          },
          required: [ 'id', 'title', 'content' ]

        let(:id) { Blog.create(title: 'foo', content: 'bar').id }
        run_test!
      end
      ```It will add schema example